docs/swagger.json

```
@@ -2296,6 +2363,32 @@
   },
   "components": {
     "schemas": {
+      "payment": {
+        "type": "object",
+        "properties": {
+          "value": {
+            "type": "object",
+            "properties": {
+              "type": {
+                "type": "string",
+                "enum": ["cash", "cb", "tr"]
+              },
+              "value": {
+                "type": "string",
+                "example": "20"
```

The value row of the schema should be an array of document.
```suggestion
            "type": "array",
            "items": {
              "type": "object",
              "properties": {
                "type": {
                  "type": "string",
                  "enum": ["cash", "cb", "tr"]
                },
                "value": {
                  "type": "string",
                  "example": "20"
                }
```